dist_tables/dml.rst

```diff
@@ -162,3 +162,18 @@ Maximizing Write Performance
 ----------------------------
 
 Both INSERT and UPDATE/DELETE statements can be scaled up to around 50,000 queries per second on large machines. However, to achieve this rate, you will need to use many parallel, long-lived connections and consider how to deal with locking. For more information, you can consult the :ref:`scaling_data_ingestion` section of our documentation.
+
+Running a Query on All Workers
+------------------------------
+
+Broadcasting a statement for execution on all workers is useful for viewing properties of entire worker databases or creating UDFs uniformly throughout the cluster. For example:
```

Comment:
As you note, we expect this UDF to be used primarily for cluster management. Could you move this section there?